django/db/models/functions.py

```diff
@@ -137,3 +137,16 @@ class Upper(Func):
 
     def __init__(self, expression, **extra):
         super(Upper, self).__init__(expression, **extra)
+
+
+class Now(Func):
+    template = 'CURRENT_TIMESTAMP()'
```

Comment:
Actually removing the trailing parentheses (e.g. `CURRENT_TIMESTAMP`) should work on all backends including SQLite.